pandas/tests/test_format.py

```diff
@@ -2343,7 +2343,21 @@ def test_csv_to_string(self):
         df = DataFrame({'col' : [1,2]})
         expected = ',col\n0,1\n1,2\n'
         self.assertEqual(df.to_csv(), expected)
-
+        
+    def test_to_csv_decimal(self):
+        df = DataFrame({'col1' : [1], 'col2' : ['a'], 'col3' : [10.1] })
```

Comment:
There should be something special that I do to create my local version...

When I do the command "git checkout master" in my to level directory I get
    error: pathspec 'master' did not match any file(s) known to git.

If I do it in the "pandas" directory (the one with the setup.py file) I get:
   Already on 'master'
   Your branch is up-to-date with 'origin/master'.
but then the second command git pull pandas/master master lead to
    fatal: 'pandas/master' does not appear to be a git repository
    fatal : Could not read from remote repository


By the way, in my repository, I don't think I've made any branch. I've done
everything in master. Maybe was that not a good idea ?



On Tue, Mar 3, 2015 at 9:31 AM, Stephan Hoyer <notifications@github.com>
wrote:

> In pandas/tests/test_format.py
> <https://github.com/pydata/pandas/pull/8448#discussion_r25670074>:
>
> > @@ -2343,7 +2343,21 @@ def test_csv_to_string(self):
> >          df = DataFrame({'col' : [1,2]})
> >          expected = ',col\n0,1\n1,2\n'
> >          self.assertEqual(df.to_csv(), expected)
> > -
> > +
> > +    def test_to_csv_decimal(self):
> > +        df = DataFrame({'col1' : [1], 'col2' : ['a'], 'col3' : [10.1] })
>
> Try this:
>
> git checkout master
> git pull pandas/master master
> git checkout your-branch
> git rebase master
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydata/pandas/pull/8448/files#r25670074>.
>



-- 
Bertrand Haut